docs/user-guide/zonal-average.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>This first plot of the global data is taken a valuable space that can better be used for the section below. Myayb e this plot should be moved to just before the first zonal avg plot, also providing some brief intro to it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>